doc/run-models.md

```diff
@@ -0,0 +1,107 @@
+# How to produce population projections
+
+## Introduction
+`witan.models.demography` is a Clojure library to run demographic models.
+
+The aim of `witan.models.demography` is to offer "generic" models in the sense that our models will be adaptable to different methodologies and data inputs. To learn more about the methodology, see the [modelling docs](intro.md). To learn more about the data inputs currently being used, jump to the [data inputs](#data-inputs) section.
+
+A minimal version of the population projection is now available for English unitary authorities, non-metropolitan districts, and boroughs. Projections are also available for the countries of Northern Ireland, Scotland, and Wales. 
+
+## Running a population projection
+
+An executable.jar file, which contains all the code and data needed to produce a projection, can be downloaded from http://example.com/FIXME.
+
+The .jar file contains the following:
```

Comment:
Saying the jar "contains" the input and output sounds a bit weird, doesn't need changing, just highlighting